### PR TITLE
Added Escher in a moment of lack of ideas

### PR DESCRIPTION
**Pull request by floridop**
Added:
  - Some text about artistic block
  - A random ASCII art

My ideas are getting definitely less interesting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,74 @@
 # MNXB01-learn
 Repo used during the MNXB01 course at LU 2021 fall
+
+It was rather hard to get inspired by such a correct statement so
```
Comment (AnttonLA) [inline README.md:4]: Testing out the line-specific comments, and also taking the chance to apologize about my uninspiring statement:)

**Comment by AnttonLA** [comment]
Hello Florido!
Thank you for the nice ASCII art. As a follow-up, could I ask you to also add a link to an image of the original Escher painting?
Thanks!
-Antton

File: README.md (hunk 1 above)
Comment (floridop) [reply README.md:4]: No need to apologize, correctness is nice :)

**Comment by floridop** [comment]
Request addressed in commit 55d0b95 

Anything else?

**Review by AnttonLA — APPROVED** [review]
Thank you! Approving changes through the review functionality before merging

**Comment by AnttonLA** [comment]
> Request addressed in commit [55d0b95](https://github.com/AnttonLA/MNXB01-learn/commit/55d0b954b5a752cfc4cafabf76e8d06fa68fd958)
> 
> Anything else?

I can't really think of anything else to ask for. Merging pull request now:)

**Pull request merged**